web/src/components/__tests-data__/columns.tsx

```diff
@@ -1,7 +1,7 @@
-import { Column, getDefaultColumns } from '../../utils/columns';
 import * as _ from 'lodash';
+import { Column, getDefaultColumns } from '../../utils/columns';
 
 // Customize columns order
-const columns = getDefaultColumns();
+const columns = getDefaultColumns(k => k);
```

Comment:
In that case I would set default value with `(k: string) => k` and keep argument only in `NetflowTraffic` component :
`export const getDefaultColumns = (t: TFunction = (k: string) => k): Column[] => {`

Reply:
hmm i'm not sure, because it isn't meant to be unset in the nominal case. It's more here in tests that is a special case where we override that i18n TFunction, but in any other usage, it must be provided

Reply:
ok that makes sense